docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.4'
```

Comment:
Wir haben bereits einen Docker-Setup in einem separaten Repo, ich möchte das lieber nicht vermischen oder nochmals einen weiteren Docker Setup starten...